APMrover2/RC_Channel.cpp

```diff
@@ -0,0 +1,266 @@
+#include "Rover.h"
+
+#include "RC_Channel.h"
+
+const AP_Param::GroupInfo RC_Channels::var_info[] = {
```

Comment:
AH! Why do vehicles have to own a copy of the var table? That will make it extremely hard to cope with extending the number of RC channels without also remembering to edit every vehicle. (Otherwise we will end up with an RC_Channel that doesn't have it's params set).